main.js

```diff
@@ -35,4 +35,165 @@ menuCon.addEventListener('click', () => {
+    ulList.appendChild(cat);
+  });
+  firstSection.appendChild(ulList);
+  firstSection.innerHTML += `<div class="paragraph">
+    <h4>
+        ${projectSection[i].description}
+    </h4>
+  </div>
+`;
+  firstSection.classList.add('first-section-info');
+  subsection.append(firstSection);
+  const listItems = document.createElement('ul');
+  listItems.classList.add('btn-changes');
+
+  projectSection[i].technologies.forEach((elem) => {
+    const list = document.createElement('li');
+    listItems.appendChild(list);
+    const buttons = document.createElement('button');
+    list.appendChild(buttons);
+    buttons.textContent = elem;
+  });
+  firstSection.appendChild(listItems);
+  firstSection.innerHTML += `<div class="big-btn">
+  <button type="button" class="btn-4">See Project</button>
+</div>`;
+  workSection.appendChild(subsection);
+}
+
+const seeMore = document.querySelectorAll('.btn-4');
+seeMore.forEach((element, index) => {
+  element.addEventListener('click', () => {
+    const popUpWin = document.createElement('div');
+    popUpWin.classList.add('details-modal');
+    const mainPopUpWin = document.createElement('div');
+    mainPopUpWin.classList.add('sub-modal');
+    popUpWin.appendChild(mainPopUpWin);
+    document.body.appendChild(popUpWin);
+    mainPopUpWin.innerHTML += `
+    <button class="close-pop">
+        <i class="fa-solid fa-xmark"></i>
+    </button>
+<div>
+    <h3 class="pop-heading">
+    ${projectSection[index].name}
+    </h3>
+</div>
+`;
+    const ulList = document.createElement('ul');
+    ulList.classList.add('list-items');
+
+    projectSection[index].category.forEach((elem, i) => {
+      const cat = document.createElement('li');
+      cat.className = `item-${i}`;
+      cat.textContent = elem;
+      ulList.appendChild(cat);
+    });
+    mainPopUpWin.appendChild(ulList);
+    mainPopUpWin.innerHTML += `
+            <div class="pop-img">
+                <img src="${projectSection[index].featuredImage}" alt="" class="responsive">
+            </div>
+            <div class="modal-para">
+            <p class="pop-message">
+                ${projectSection[index].description}
+            </p>
+            </div>
+`;
+    const techList = document.createElement('ul');
+    techList.classList.add('pop-tech');
+
+    projectSection[index].technologies.forEach((elem, index) => {
+      const list = document.createElement('li');
+      techList.appendChild(list);
+      const buttons = document.createElement('button');
+      buttons.classList.add(`btn-${index + 1}`);
+      list.appendChild(buttons);
+      buttons.textContent = elem;
+    });
+    mainPopUpWin.appendChild(techList);
+    mainPopUpWin.innerHTML += `
+<div>
+    <ul class="pop-button">
+         <li><button type="button" class="pop-btn">See live <img src="images/Icon1.svg" alt=""></button></li>
+          <li><button type="button" class="pop-btn">See Source <img src="images/Vector2.svg" alt=""></button></li>
+    </ul>
+</div>
+`;
```

Comment:
- [ ] Currently, your pop-ups seem to have a bug as the text in the background is visible. It might be because the blur effect was implemented on the wrong container. Kindly fix this, and make sure you conform to the Figma template in both the mobile and desktop version (Even the background blur)
> Yours 
![image](https://user-images.githubusercontent.com/90556221/169508306-77af103a-2bfe-4703-9e3a-777234e9a032.png)
![image](https://user-images.githubusercontent.com/90556221/169508511-c92d8b47-eb01-40a2-af31-fa2ee688a5dd.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/169508774-821de5c6-c43a-4b62-bf87-d63710dcff83.png)

![image](https://user-images.githubusercontent.com/90556221/169508701-6dbd44c9-27d9-4ef7-9c8f-f7381444acfb.png)

- [ ] Please *ensure that you follow* the Figma design as accurately as possible, as you can see in the images above, the designs are entirely different. Especially for the desktop version where the buttons are placed wrongly and the title and the sub-heading have the wrong font weights and sizes.
- [ ] Please make sure that the Pop up is centered on the screen.